### ISSUE TITLE
Demo Bug

### ISSUE THREAD

**Issue by RobPruzan** [HSLCY/GlossBERT#11]
F.softmax is being called on the SequenceClassifierOutput object, not the tensor
